backend/events/migrations/0005_auto_20201128_2048.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.1.3 on 2020-11-28 20:48
```

Comment:
Rename the migration to a meaningful name instead of `auto_20201128_2048`

Reply:
Done.